app/components/tags_input/tags_input.rb

```diff
@@ -2,24 +2,24 @@
 
 module TagsInput
   class TagsInput < ApplicationComponent
-    def initialize(value: [], options: [], **kwargs)
+    def initialize(value: [], available_tags: [], allow_new: true, **kwargs)
```

Comment:
```suggestion
    def initialize(value: [], allow_new: true, **kwargs)
```
can be removed now, right?

Reply:
That’s actually the only thing I didn't like about your fix. I think the TagsInput view component should not rely on business logics/concrete models